.github/workflows/build.yaml

```diff
@@ -0,0 +1,26 @@
+
+# SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+name: Build CONDA PKG
+
+on:
```

Comment:
Do we want to setup a nightly build?

Reply:
The cadence of development here will probably be slower than pynjitlink, for example.  So I don't think we need nightly builds (at least not yet).  Instead, I think we should manually do release for a couple reasons:
1. Let's effort now and quicker to ship an artifact to NeMo folks
2. We should learn a bit more about the NVIDIA channel before uploading nightly

Again, we can draw for pynjitlink for managing releases -- I think it relies on GH actions and manual tagging for uploading packages:
- https://github.com/rapidsai/pynvjitlink/pull/74